### PR TITLE
feat: made the link button

### PR DESCRIPTION
**Pull request by therajatg**

### CODE REVIEW

**Review by parulgupta2001 — APPROVED** [review]
For the Secondary Link, you gave class name as secondary-link but applied style on secondary-link-btn.

File: styles/component-styles/button.css [hunk 1]
```diff
@@ -1,68 +1,78 @@
...
+.secondary-link-btn {
+  color: var(--delete-btn-color);
+  font-weight: 600;
+}
+
+.floating-action-button {
+  background-color: var(--primary-color);
+  display: flex;
+  justify-content: center;
+  align-items: center;
```
Comment (srejitk) [inline styles/component-styles/button.css]: Would suggest using utility classes here

File: styles/component-styles/button.css [hunk 2]
```diff
@@ -1,68 +1,78 @@
...
+  border-radius: 999rem;
```
Comment (srejitk) [inline styles/component-styles/button.css]: Avoid keeping such high values for border radius.

File: styles/component-styles/button.css [hunk 3]
```diff
@@ -1,68 +1,78 @@
...
+  width: 2.5rem;
+  height: 2.5rem;
+}
+
+.floating-action-button a {
+  font-size: xx-large;
```
Comment (srejitk) [inline styles/component-styles/button.css:73]: Try to keep a consistent styling, either use text or size in px as you've done above.

**Review by srejitk — APPROVED** [review]
Some Minor Suggestions. Looks good otherwise.

File: styles/component-styles/button.css (hunk 1 above)
Comment (therajatg) [reply styles/component-styles/button.css]: Good suggestion. Done

File: styles/component-styles/button.css (hunk 2 above)
Comment (therajatg) [reply styles/component-styles/button.css]: made border radius 50%

Comment (therajatg) [reply styles/component-styles/button.css]: and made a different class for it

**Pull request merged**